github-sync/github-data/sigstore/repositories.yaml

```diff
@@ -1459,6 +1459,8 @@ repositories:
     hasIssues: true
     hasProjects: false
     hasWiki: false
+    pages:
+      buildType: workflow
```

Comment:
This doesn't exist as an option under https://github.com/sigstore/github-sync/blob/201b23090cb3cbc628dc256d683d02342792a629/pkg/config/config.go#L59 - is your use of pages different than our other repos that use it?

Reply:
yes it is different, see the github-sync link in the PR message.

I maybe shouldn't have marked this "ready for review", sorry (but with two PRs keeping this a draft until the other is merged does likely mean a multi-day review cycle) .

Reply:
ah, my bad - i missed that. approved and merged that PR.